e2e/cypress/integration/messaging/message_reply_part2_spec.js

```diff
@@ -0,0 +1,155 @@
+
+        // * RHS is open
+        cy.get('#rhsContainer').should('be.visible');
+
+        // * `Add Comment` button is disabled
+        cy.get('#addCommentButton').should('be.disabled');
+
+        // # Type a character in the comment box
+        cy.get('#reply_textbox').type('A');
+
+        // * `Add Comment` button is not disabled
+        cy.get('#addCommentButton').should('not.be.disabled');
+
+        // # Clear comment box
+        cy.get('#reply_textbox').clear();
+
+        // # Close RHS
+        cy.closeRHS();
+    });
+
+    it('MM-T2134 - Reply to message displays in RHS and center and shows reply count', () => {
+        // # Open RHS comment menu
+        cy.clickPostCommentIcon(rootId);
+
+        const msg = 'reply1';
+
+        // # Type message
+        cy.get('#reply_textbox').type(msg);
+
+        // # Post reply
+        cy.get('#addCommentButton').click();
+
+        cy.getLastPostId().then((replyId) => {
+            // * Message displays in center
+            cy.get(`#post_${replyId}`).within(() => {
+                cy.get(`#postMessageText_${replyId}`).should('be.visible').and('have.text', msg);
+            });
+
+            // * Message displays in RHS
+            cy.get(`#rhsPost_${replyId}`).within(() => {
+                cy.get(`#rhsPostMessageText_${replyId}`).should('be.visible').and('have.text', msg);
+            });
+
+            // * Reply count is visible and shows expected value
+            cy.get(`#CENTER_commentIcon_${rootId} .post-menu__comment-count`).should('be.visible').and('have.text', '1');
+        });
+
+        // # Close RHS
+        cy.closeRHS();
+    });
+
+    it('MM-T2135 - Can open reply thread from reply count arrow and reply', () => {
+        // # Click reply icon
+        cy.get(`#CENTER_commentIcon_${rootId}`).click().wait(TIMEOUTS.HALF_SEC);
+
+        const msg = 'reply2';
+
+        // # Type message
+        cy.get('#reply_textbox').type(msg);
+
+        // # Press `Enter`
+        cy.get('#reply_textbox').type('{enter}');
+
+        cy.getLastPostId().then((replyId) => {
+            // * Message displays in center
+            cy.get(`#post_${replyId}`).within(() => {
+                cy.get(`#postMessageText_${replyId}`).should('be.visible').and('have.text', msg);
+            });
+
+            // * Message displays in RHS
+            cy.get(`#rhsPost_${replyId}`).within(() => {
+                cy.get(`#rhsPostMessageText_${replyId}`).should('be.visible').and('have.text', msg);
+            });
+        });
+
+        // # Close RHS
+        cy.closeRHS();
+    });
+
+    it('MM-T2136 - Reply in RHS with different channel open in center', () => {
+        // # Click on the `...` menu icon and click on `Reply`
+        cy.uiClickPostDropdownMenu(rootId, 'Reply', 'CENTER');
+
+        // # Switch to a different channel
+        cy.get(`#sidebarItem_${otherChannel.name}`).click().wait(TIMEOUTS.FIVE_SEC);
```

Comment:
Does this fail under 5 seconds? It seemed quite long to me 😅

Reply:
Not sure but I've seen it used when switching channels on other tests and ported it here. It's to make sure all state has been properly loaded.